src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Parcela.kt

```diff
@@ -1,35 +1,34 @@
 package ar.edu.unahur.obj2.semillasAlViento
 
-class Parcela(val ancho: Int, val largo: Int, val horasSolPorDia: Int) {
+class Parcela(private val ancho: Int, private val largo: Int, private val horasSolPorDia: Int) {
+
   val plantas = mutableListOf<Planta>()
-  var cantidadPlantas = 0
 
   fun superficie() = ancho * largo
+
   fun cantidadMaximaPlantas() =
     if (ancho > largo) ancho * largo / 5 else ancho * largo / 3 + largo
```

Comment:
Acá podrían haber reemplazado el código repetido por un llamado al método `superficie()`:
```suggestion
    if (ancho > largo) superficie() / 5 else superficie() / 3 + largo
```